conf/Corefile.example

```diff
@@ -0,0 +1,21 @@
+.:54 {
```

Comment:
That file should likely be in conf/caddy-services/pfdns.conf.example

(Note that I'm assuming we'll be calling it pfdns in v8 which may be wrong)